Lil-Peep/src/sections/featured-kits.liquid

```diff
@@ -0,0 +1,160 @@
+              </div>
+       {% else %}
+        {% if block.settings.opc == "4" %}
+          <div class="mid-banner mid-banner-archeologist banner-{{block.id}}">
+            <div class="mid-banner-content">
+              <h1>{{ block.settings.title }}</h1>
+              <p>{{block.settings.description}}</p>
+              <button type="button" name="button"><h1>{{block.settings.button}}</h1></button>
+            </div>
+          </div>
+        {% endif %}
+      {% endif %}
+    {% endfor %}
+  </div>
+</section>  }
+
+<!-- Styles for the background of the 2 mid-banner -->
+<style>
+{% for block in section.blocks %}
+  .banner-{{block.id}} {
+    background-image: url('{{ block.settings.banner_img | img_url: 'large'}}');
+    background-size: cover;
+  }
+  {% if block.settings.opc == "3" %}
+  .mid-banner-saylor {
+    background: linear-gradient(0deg,
+      rgb(239, 239, 239),
+      rgba(239, 239, 239, 1) 20%,
+      rgba(239, 239, 239, 0.2) 50%,
+      rgba(239, 239, 239, 0)),url("{{ block.settings.banner_img | img_url: 'large'}}");
+    background-size: cover;
+    color: #000000;
+  }
+  {% else %}
+    {% if block.settings.opc == "4" %}
+     .mid-banner-archeologist {
+        background: linear-gradient(0deg,
+          rgb(189, 152, 107),
+          rgba(189, 152, 107, 1) 10%,
+          rgba(189, 152, 107, 0.2) 50%,
+          rgba(189, 152, 107, 0),url("{{ block.settings.banner_img | img_url}}"));
+        background-size: cover;
+        color: #FFFFFF;
+      }
+    {% endif %}
+  {% endif %}
+{% endfor %}
+</style>
+
+
+{% schema %}
+{
+ "name":"Featured Kits",
+ "max_blocks":4,
+ "blocks":[
+   {
+     "type":"image",
+     "name":"Kit",
+     "settings":[
+       {
+         "type":"image_picker",
+         "id":"banner_img",
+         "label":"Image",
+         "info":"image should be 800px x 800px"
+       },
+       {
+         "type": "text",
+         "id": "title",
+         "label": "Title",
+         "default": "Title"
+       },
+       {
+         "type": "text",
+         "id": "description",
+         "label": "Description",
+         "default": "Description"
+       },
+       {
+         "type": "text",
+         "id": "button",
+         "label": "Button",
+         "default": "Text"
+       },
+       {
+         "type":      "select",
+         "id":        "opc",
+         "label":     "Type of banner",
+         "options": [
+           {
+             "group": "Top-Banners",
+             "value": "1",
+             "label": "Big banner"
+           },
+           {
+             "group": "Top-Banners",
+             "value": "2",
+             "label": "Small banner"
+           },
+           {
+             "group": "Low-Banners",
+             "value": "3",
+             "label": "Left banner"
+           },
+           {
+             "group": "Low-Banners",
+             "value": "4",
+             "label": "Right banner"
+           }
+         ],
+         "default":   "1",
```

Comment:
Aqui tambien hay espacio entre los atributos

Reply:
I think it is used to improve readability of the code.